apps/admin-page/src/app/app.component.html

```diff
@@ -1,5 +1,79 @@
-<div>
-    <span>Angular Material: </span>
-    <mat-slider min="1" max="100" step="1" value="50"></mat-slider>
-    <router-outlet></router-outlet>
-</div>
+<!-- Navbar -->
+<mat-toolbar color="primary" data-cy="navbar">
+    <mat-toolbar-row>
+        <span>TC management</span>
```

Comment:
The logo/name? should redirect to the home page
```suggestion
        <a mat-button routerLink="/">TC management</a>
```
_mat-button is used to apply material styling which overrides default `a` element styling_

Reply:
We have decided to include this change when other frontend issues are worked on
Resolved